modules/bitgo/src/v2/coinFactory.ts

```diff
@@ -179,7 +179,7 @@ import {
   ZkethToken,
 } from './coins';
 
-function registerCoinConstructors(globalCoinFactory: CoinFactory): void {
+export function registerCoinConstructors(globalCoinFactory: CoinFactory, coinMap: CoinMap = coins): void {
```

Comment:
because we are not using the [GlobalCoinFactory](https://github.com/BitGo/BitGoJS/blob/master/modules/bitgo/src/v2/coinFactory.ts#L424) now
```suggestion
export function registerCoinConstructors(coinFactory: CoinFactory, coinMap: CoinMap = coins): void {
```